emperor/support_files/js/color-view-controller.js

```diff
@@ -132,16 +132,26 @@ define([
           var data = decompViewDict.setCategory(attributes, scope.setPlottableAttributes, category);
 
           if (scaled) {
-            scope.setSlickGridDataset({});
-            scope.$gridDiv.hide();
+            //build plottables as all non-numerics
```

Comment:
I think this whole block about determining non-numeric values and their respective plottables belongs in a separate function that we should be able to test, the edge cases here sound like a nightmare to have untested.